### ISSUE TITLE
Existing CKAN Install using Cookie Based Session Breaks with 7406/7464

### ISSUE THREAD

**Issue by JVickery-TBS** [ckan/ckan#7651]
## CKAN version
2.10 / master branch - latest

## Describe the bug
Fatal error on non-existant `beaker.session.validate_key`

### Steps to reproduce
* Have an existing 2.10 / master branch CKAN install from pre https://github.com/ckan/ckan/pull/7464 merge;
* Have that CKAN install be using cookie based sessions;
* Pull latest master (upgrade install to latest master);
* Reboot/Start CKAN process;
* See fatal error.

### Expected behavior
Users does not get a fatal error, and the `beaker.session.validate_key` should be migratable.

### Additional details
From the 7464 merge with changelog for issue https://github.com/ckan/ckan/issues/7406

```
Important Note:

To ensure proper functionality when using cookie-based sessions, it is now required to set `beaker.session.validate_key` appropriately.
```

**Comment by smotornyuk** [comment]
Will be fixed by #7450

**Comment by JVickery-TBS** [comment]
@smotornyuk is this one solved? I see merges from https://github.com/ckan/ckan/pull/7450 and https://github.com/ckan/ckan/pull/7781 but now entirely sure as https://github.com/ckan/ckan/pull/7450 was just closed??

**Comment by smotornyuk** [comment]
Yes, Adria got Tome's work from #7450 and added more fixes in #7781. `validate_key` now has the [configured_default validator](https://github.com/ckan/ckan/blob/ckan-2.10.7/ckan/config/config_declaration.yaml#L321) that internally copies value of the `beaker.session.secret` when `validate_key` is empty. This fix is available in the latest version of v2.10

**Comment by smotornyuk** [comment]
Logic for secrets consolidated in #7781
